docs/root/intro/arch_overview/http_connection_management.rst

```diff
@@ -60,6 +60,9 @@ can be used to modify this behavior, and they fall into two categories:
   * *envoy.retry_host_predicates.previous_hosts*: This will keep track of previously attempted hosts, and rejects
     hosts that have already been attempted.
 
+  * *community.retry_host_predicates.omit_canary_hosts*: This will ensure that a retry will always be done on
```

Comment:
Since this extension lives in the envoy repo we'll want to prefix the name with `envoy` like all the other extensions